pkg/authorizer/authorizer.go

```diff
@@ -4,6 +4,7 @@ import (
 	"log"
 
 	"github.com/awslabs/aws-eks-cluster-controller/pkg/apis"
+	extapi "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
MINOR: if you rename this import - lets call it v1beta1apis or something

Reply:
Done